addon/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as ContainerQuery } from './components/container-query';
+export { default as aspectRatio } from './helpers/cq-aspect-ratio';
+export { default as height } from './helpers/cq-height';
+export { default as width } from './helpers/cq-width';
```

Comment:
**questions**

1\. Can you clarify for me how the end-developer would consume these exported objects? (If you can show me a sample code, that'd help me understand.)

2\. Could we rename the objects so that the names match those in the respective files?

```ts
export { default as ContainerQueryComponent } from './components/container-query';
export { default as cqAspectRatio } from './helpers/cq-aspect-ratio';
export { default as cqHeight } from './helpers/cq-height';
export { default as cqWidth } from './helpers/cq-width';
```

3\. When you rebase your branch, the latest commit will show that there is now the `{{container-query}}` modifier.

Would it make sense to export the modifier in `addon/index.ts` as well? (I'm not sure yet how having `addon/index.ts` helps end-developers—see question 1 above.)

```ts
export { default as ContainerQueryComponent } from './components/container-query';
export { default as cqAspectRatio } from './helpers/cq-aspect-ratio';
export { default as cqHeight } from './helpers/cq-height';
export { default as cqWidth } from './helpers/cq-width';
export { default as ContainerQueryModifier } from './modifiers/container-query`;
```

Reply:
Thanks for a prompt response. That's very interesting!

Let me ponder over the names more over the weekend and get back to you early next week.

Reply:
I included this change in a recent patchfile I made so I could continue upgrading https://limber.glimdown.com https://github.com/NullVoxPopuli/limber/pull/621/files#diff-b2a12a590e7e29c8f846dbc02ddc7973ccd718121f8bda6e44ca9658aeb2a650

(I'll be updating _usage_ of my patched ember-container-query in a future PR)

:tada: